core/src/main/java/org/apache/any23/writer/LoggingTripleHandler.java

```diff
@@ -111,6 +116,6 @@ public void setContentLength(long contentLength) {
     }
 
     private void printHeader(PrintWriter writer) {
-        writer.println("# Document-IRI\tContent-Length\tElapsed-Time\tSuccess\tExtractors");
+        writer.println("# Document-IRI\tContent-Length\tElapsed-Time\tSuccess\tExtractors:Triples");
```

Comment:
Hi,

Yes that is the intention. That behaviour was implemented in the `LoggingTripleHandler` and was displayed but I have just added header for that. For example in last `[ yaml:531 ]` in the logging file only. That information was not displayed in *strerr* output.